### PR TITLE
Disability Settings - Curable Option - Second Attempt

### PR DESCRIPTION
**Pull request by Kiyahitayika**
## What Does This PR Does:
In short, this PR aims to address: https://github.com/ScorpioStation/ScorpioStation/issues/274
- Prevents curing via the CureDisability procs 
- Prevents curing via Admemery - same as above
- Prevents curing via DNA SubBlock radiation done through the DNA modifier machine

## Requires:

PR: #1 
PR: #2  

## Why It's Good For The Game
When someone selects a Disability for their character in Setup, they may not want that Disability to be curable. Not everything needs to be cured. Difference is beautiful.

## Images of changes

image

## Changelog
:cl:
add: Curability Option toggle on Disabilities in Character Setup. Choosing "No" makes associated Disability incurable for entire gameround.
/:cl:

### CODE REVIEW

File: code/game/dna/genes/goon_disabilities.dm [hunk 1]
```diff
@@ -40,13 +24,7 @@
...
 
 /datum/dna/gene/disability/radioactive/can_activate(mob/M, flags)
-	if(!..())
-		return FALSE
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if((RADIMMUNE in H.dna.species.species_traits) && !(flags & MUTCHK_FORCED))
-			return FALSE
-	return TRUE
+	radiation_pulse(H, 20)
```
Comment (Kiyahitayika) [inline code/game/dna/genes/goon_disabilities.dm:27]: . . . why. brain.

**Comment by Kiyahitayika** [comment]
👀 Whoops. I was trying to merge this into my Workshop Repo. But, uh .... hey, @blinkdog ? It only ran icondiffbot now. Is that...is that right? .__.;

**Comment by Kiyahitayika** [comment]
Yeah, please ignore this PR, it wasn't supposed to be opened. Also, by some code out of the goon_disabilities.dm file, that's not license-okay. .__.

**Comment by blinkdog** [comment]
> eyes Whoops. I was trying to merge this into my Workshop Repo. But, uh .... hey, @blinkdog ? It only ran icondiffbot now. Is that...is that right? .__.;

Yes, it's right. :smile_cat:

I removed the integration between the ScorpioStation repository and MapDiffBot.
I replaced it with our own GitHub Action to generate map change images (See #343).

GitHub hard caps us at 0.5 GB and hitting that limit won't affect anyone but us.
Fixing that is just deleting the old map change images we no longer need to make more room.
Simple and safe; no worries on any map changes going forward.

**Pull request closed without merging**